addon/components/object-list-view.js

```diff
@@ -31,6 +31,18 @@ export default FlexberryBaseComponent.extend(
   */
   _modelProjection: null,
 
+  /**
+    Flag indicates whether to look for changes of model (and displaying corresponding changes on control) or not.
+
+    If flag enabled component compares current detail array with used on component,
+    removes deleted and marked as deleted on model level records, adds created on model level records.
+
+    @property searchForContentChange
+    @type Boolean
+    @default false
+  */
+  searchForContentChange: false,
```

Comment:
I think that it is default behavior for ember components.
They absolutely should react on binded data changes and display them without any flags and other settings.
Another behavior is not the ember way.
So searchForContentChange is unnecessary setting.